src/status_im2/common/standard_auth/view.cljs

```diff
@@ -0,0 +1,154 @@
+(ns status-im2.common.standard-auth.view
+  (:require [react-native.core :as rn]
+            [quo2.components.numbered-keyboard.numbered-keyboard.view :as numbered-keyboard]
+            [quo2.core :as quo]
+            [status-im2.common.standard-auth.style :as style]
+            [reagent.core :as reagent]
+            [quo2.foundations.colors :as colors]
+            [react-native.fast-image :as fast-image]
+            [status-im2.common.resources :as resources]
+            [utils.i18n :as i18n]
+            [quo2.components.keycard.view :as keycard]
+            [utils.re-frame :as rf]))
+
+(defn enter-keycard-pin-sheet
+  []
+  (let [entered-numbers (reagent/atom [])
+        max-digits 6
+        max-attempt-reached (reagent/atom false)]
+    (fn []
+      (defn handle-number-entry [number]
+        (when (>= (count @entered-numbers) max-digits)
+          (reset! entered-numbers []))
+        (when (= (count @entered-numbers) (dec max-digits))
+          (reset! max-attempt-reached true))
+        (swap! entered-numbers conj number))
+
+      (println "entered-numbers" @entered-numbers)
+
+      (defn get-dot-style [idx]
+        (if (<= idx (dec (count @entered-numbers)))
+          (if @max-attempt-reached
+            (assoc style/digit :backgroundColor colors/danger)
+            (assoc style/digit :backgroundColor colors/white))
+          (assoc style/digit :backgroundColor colors/neutral-50)))
+
+      [rn/view {:style style/container}
+       [rn/view {:style style/inner-container}
+        [quo/text {:accessibility-label :enter-keycard-pin-text :size :heading-2 :weight :semi-bold} "Enter Keycard PIN"]
```

Comment:
Let's move "Enter Keycard PIN" over to our translation file ([en.json](https://github.com/status-im/status-mobile/blob/0003800f052d5e6f650b6ac08e64ad550aa26739/translations/en.json)). Then, we can simply use `(i18n/label :t/<label>)`

Reply:
@ajayesivan Moved this one and also other texts to `en.json` file.